pandas/io/gbq.py

```diff
@@ -14,21 +19,21 @@ def _try_import():
 
 
 def read_gbq(
-    query,
-    project_id=None,
-    index_col=None,
-    col_order=None,
-    reauth=False,
-    auth_local_webserver=False,
-    dialect=None,
-    location=None,
-    configuration=None,
+    query: str,
+    project_id: Optional[str] = None,
+    index_col: Optional[str] = None,
+    col_order: Optional[List[str]] = None,
+    reauth: bool = False,
+    auth_local_webserver: bool = False,
+    dialect: Optional[str] = None,
+    location: Optional[str] = None,
+    configuration: Optional[Dict] = None,
```

Comment:
Can subtypes be added for Dict here?

Reply:
Added `Dict[str, Any]` since it is essentially key value pairs that are sent to the biq query api. Unsure if we can be any stricter on the values?

https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query